support.md

```diff
@@ -51,26 +51,53 @@ reported issues within 2 business days.
-the exact target date for the incoming month. Those release candidate should pass all our unit tests,
-and show no unwanted regression in the projects tested on [ci.bazel.io](http://ci.bazel.io).
+candidate for a new MINOR version (e.g. 0.6.0). The work is tracked by a [release bug on GitHub](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+is%3Aopen+label%3Arelease) which indicates
+the exact target date for the incoming month and assigned to the current Release Manager.
+Those release candidate should pass all our unit tests, and show no unwanted regression in the
+projects tested on [ci.bazel.io](http://ci.bazel.io).
 
 We announce those release candidates on [bazel-discuss](https://groups.google.com/forum/#!forum/bazel-discuss);
-these are binaries that have passed all of our unit tests. Over the next few days, we regression
-test all applicable build targets at Google. If you have a critical project using Bazel, we
-recommend that you establish an automated testing process that tracks the current release
-candidate, and report any regressions.
+these are binaries that have passed all of our unit tests. Over the next days, we monitor community bug
+reports for regression in release candidate.
 
 If no regressions are discovered, we officially release the candidate after two week. However,
 regressions can delay the release of a release candidate. If regressions are found, we apply
 corresponding cherry-picks to the release candidate to fix those regressions. If no further
 regressions are found for two business days, but not before two week has elapsed since the first
-release candidate, we release it.
+release candidate, we release it. We do not cherry-pick non-regression bugfix (e.g., for
+bug that has always been there or bug on new features) or feature.
+
+A release can only be release on a day where the next day is a business day. The release manager
+should make sure he is available on the next day or delegate its responsabilities.
+
+If a regression is found on the latest release, a patch release can be emitted by applying the
+corresponding cherry-pick to the release tag. Being another patch to an existing release, the
+release candidate for a patch release can be released after 2 business days.
+
+### Post-mortems
+
+We are doing public post-mortems in the following cases:
+
+- We are unable to create a viable release candidate 2 business days after the target day,
+- We are unable to release a candidate by the end of the month,
+- An emergency patch release is needed (a regression that cause the current binary to increase
+  the support load, e.g. several users report the same issue after a package manager update to
+  the new release).
+
+### Testing
+
+We run nightly build of all the projects running on [ci.bazel.io](http://ci.bazel.io) using Bazel
+binaries built at head, and using the release binaries. We project going to be impacted by a
```

Comment:
"We project going to be impacted by a breaking change" - I cannot understand this statement.

Reply:
wo shuo hanyu, bu dong? :) Sorry updated.